lib/Maui.GoogleMaps/Maui.GoogleMaps.csproj

```diff
@@ -41,6 +41,9 @@
 		<PackageReference Include="Xamarin.GooglePlayServices.Maps" Version="118.2.0.3" />
 	</ItemGroup>
 
+	<ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
```

Comment:
same. Dislike. And please open prs on your previous branches if u need the code that is not merged yet. it is hard to track the changes.

Reply:
no problem, If you approve the PRs in the correct order, the changes will be in the correct order and separated correctly in the PR.